projects/js-packages/components/components/donut-meter/index.tsx

```diff
@@ -40,6 +47,28 @@ export type DonutMeterProps = {
 	 * If not provided, defaults to an empty string.
 	 */
 	description?: string;
+
+	/**
+	 * Changes colors according to  description for meter.
+	 * Not visible. Used for a11y support.
+	 * If not provided, defaults to an empty string.
```

Comment:
The annotation seems not to fit the prop.

Reply:
Good catch, fixed!